src/lib/y2security/selinux.rb

```diff
@@ -145,43 +221,56 @@ def configurable?
     GETENFORCE_PATH = "/usr/sbin/getenforce".freeze
     private_constant :GETENFORCE_PATH
 
-    # Find SELinux mode by given value
+    # Returns the values for the SELinux setting from the product features
     #
-    # @note using nil means to set SELinux mode as disabled.
+    # @return [Hash{Symbol => String, Boolean, nil}] e.g., { mode: "enforcing", configurable: false }
+    #   a hash holding the defined SELinux options in the control file;
+    #   an empty object if no settings are defined
+    def product_feature_settings
+      @product_feature_settings unless @product_feature_settings.nil?
+
+      settings = Yast::ProductFeatures.GetFeature("globals", "selinux")
+      settings = {} if settings.empty?
+      settings.transform_keys!(&:to_sym)
+
+      @product_feature_settings = settings
+    end
+
+    # Returns a CFA::Selinux object for handling the config file
     #
-    # @param id [SelinuxConfig::Mode, String, Symbol, nil] a SELinux mode or its identifier
-    # @return [Mode] the SelinuxConfig::Mode by given id or disabled is none found or nil was given
-    def find_mode(id)
-      found_mode = Mode.find(id)
+    # @return [CFA::Selinux]
+    def config_file
+      @config_file ||= CFA::Selinux.load
```

Comment:
sadly still exception is not handled. So during installation it cause this crash during save:

![selinux_failure](https://user-images.githubusercontent.com/478871/107685871-d2159c00-6ca4-11eb-8d07-6b0e5aa07ae7.png)

Reply:
Adressed in https://github.com/yast/yast-security/pull/89, thanks for catching.